### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by pts-jchin**
- clarify instructions for gotcha
- add Java as implementation options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Please *do not fork or submit pull requests* to this repository.
 
 Please timebox your solution to one day total.
 
-* Implement your solution using JavaScript or Python.
+* Implement your solution using JavaScript or Python/Java.
```
Comment (jweatherby) [inline README.md:44]: We don't want Java on ours, so let's not be so specific. How about 
`Implement your solution using the language agreed upon with the hiring manager`

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Please timebox your solution to one day total.
 * Target the latest stable version of Google Chrome.
 * Format the code according to the latest eslint specs
 
-**If using Python**
+**If using Python or Java**
```
Comment (jweatherby) [inline README.md:55]: Instead: `If using a backend language`

File: README.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ Please timebox your solution to one day total.
...
 
 * Implement a simple yet intuitive command line interface.
-* Format the code following PEP8 guidelines
\ No newline at end of file
+* Format the code following PEP8 guidelines (Python only)
```
Comment (jweatherby) [inline README.md:58]: `(If using Python)`

File: README.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ There are different levels to this assignment, level 1 being the easiest, level
 
 * Fetch the tax rates by year: http://localhost:5000/tax-calculator/brackets/<2019|2020|...>
 
-_Warning: this endpoint has gotchas_
+_Warning: this endpoint has gotchas, which require special handling in your code._
```
Comment (jweatherby) [inline README.md:25]: How about: 
`Warning: This endpoint sometimes returns bad responses, handle as appropriate.`

**Comment by pts-jweatherby** [comment]
Will incorporate into https://github.com/Points/interview-test-server/pull/4

**Pull request closed without merging**